### PR TITLE
spice-gtk 0.42

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

~`resource` blocks may require updates.~

### CODE REVIEW

**Comment by cho-m** [comment]
Due to https://gitlab.freedesktop.org/spice/spice-gtk/-/commit/795318c8fad0288d86bd6a392c0f82be60d4b9da, build is now using shebang from script https://gitlab.com/keycodemap/keycodemapdb/-/blob/master/tools/keymap-gen#L1
```
#!/usr/bin/python3
```

This is not available on our Linux runner. May be better to get upstream to use `#!/usr/bin/env python3` instead to avoid hardcoded path.

**Comment by dawidd6** [comment]
https://gitlab.com/keycodemap/keycodemapdb/-/merge_requests/18

**Comment by chenrui333** [comment]
Thanks @dawidd6!

File: Formula/spice-gtk.rb [hunk 1]
```diff
@@ -62,6 +61,8 @@ class SpiceGtk < Formula
     sha256 "2b020ecf7d21b687f219b71ecad3631f644a47f01403fa1d1036b0c6416d70fb"
   end
 
+  patch :DATA
```
Comment (carlocab) [inline Formula/spice-gtk.rb:64]: Would be nicer to use the merged commit: https://gitlab.com/keycodemap/keycodemapdb/-/commit/c2c09b3f4c1d7688711e6501338ac88017d4341c

At minimum, we should link to the MR.

**Pull request closed without merging**